core/idl/wallet/ethereum/erc20.djinni

```diff
@@ -1,59 +1,72 @@
 @import "../../math/big_int.djinni"
 @import "../../ethereum/addresses.djinni"
-@import "../amount.djinni"
 
 # Representation of ERC20Tokens.
 ERC20Token = record {
     # Name of the currency.
-	name: string;
+    name: string;
     # Symbol representation of the currency.
-	symbol: string;
+    symbol: string;
     # Address of the smart contract for this ERC20.
-	contractAddress: string;
+    contractAddress: string;
     # Maximum number of decimals (used to show the human representation of the currency).
-	numberOfDecimal: i32;
+    numberOfDecimal: i32;
+}
+
+# ERC20-like balance history.
+ERC20LikeBalanceHistory = record {
```

Comment:
What is the purpose of this record?
The client knows first three parameters anyway, so he is only interested in the list

Reply:
Initially, this was intended to handle missing information and clamping (i.e. when you ask for something that is not available, the algorithm should clamp back). Though I will remove it as it complicates types for a situation people might not be interested in.